.github/workflows/test.yml

```diff
@@ -16,9 +16,9 @@ jobs:
         uses: actions/setup-node@v1
         with:
           node-version: ${{ matrix.node-version }}
-      - run: npm install
-      - run: npm run dev
-      - run: npm run lint
-      - run: npm test
+      - run: yarn
```

Comment:
We should use `--frozen-lockfile` here—it will fail if something needs to change.